Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -1337,8 +1345,48 @@ void MainWindow::showSceneContextMenu(const QPoint& p) {
         QMenu menu;
         Q_FOREACH(QString name, menu_actions.keys())
         {
-          QAction* action = menu.addAction(name);
-          connect(action, &QAction::triggered, this, &MainWindow::propagate_action);
+          if(name == QString("alpha slider"))
+            continue;
+          if(name == QString("Alpha value"))
+          {
+            QWidgetAction* sliderAction = new QWidgetAction(&menu);
```

Comment:
Missing `#include <QWidgetAction>` in `MainWindow.cpp`. Does it really compile on your machine? It fails on all test platforms:
https://cgal.geometryfactory.com/CGAL/testsuite/results-4.13-Ic-62.shtml#Polyhedron_Demo

Reply:
Yes it really compiles on my machine, I wonder why. I guess my include tree in built differently or something.